presto-verifier/src/main/java/com/facebook/presto/verifier/framework/Column.java

```diff
@@ -49,13 +49,21 @@
     private static final Set<Type> FLOATING_POINT_TYPES = ImmutableSet.of(DOUBLE, REAL);
 
     private final String name;
+    private final Expression expression;
     private final Category category;
     private final Type type;
 
     @VisibleForTesting
     public Column(String name, Category category, Type type)
+    {
+        this(name, delimitedIdentifier(name), category, type);
+    }
```

Comment:
The constructor is no longer needed and should be deleted as callers may used the newly introduced factory constructor `create`.